02-Logging/02-Writing-to-a-File.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>A feature suggestion for GSD, it would be nice to include a short helper function that recursively converts this flat dict with long keys to a nested dict for user analysis. Accessing all the <code>ThermodynamicQuantities</code> in one analysis script and not touching any other loggables would be a common use case, for instance to plot the total energy of the system. Not in scope for now, but would be something to take note of and see if there's interest.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Yes, we could do this if people are interested. We could also consider implementing helpers to convert logged data into pandas data frames.</p>

Reply:
Yes that would also be useful. Some of that logic can probably be reappropriated from signac's `Project.to_dataframe` method.

Reply:
Feel free to open issue(s) on the GSD repository if you think this is something important to pursue.